<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GLvlView</class>
 <widget class="QMainWindow" name="GLvlView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>671</width>
    <height>515</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QFrame" name="frame">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="lineWidth">
       <number>0</number>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>cursor</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="xCoordAim"/>
       </item>
       <item>
        <widget class="QSpinBox" name="yCoordAim"/>
       </item>
       <item>
        <widget class="QSpinBox" name="zCoordAim"/>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>93</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Camera</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="xCoordCam"/>
       </item>
       <item>
        <widget class="QSpinBox" name="yCoordCam"/>
       </item>
       <item>
        <widget class="QSpinBox" name="zCoordCam"/>
       </item>
       <item>
        <widget class="QCheckBox" name="showCursHereBtn">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>show cursor</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="glViewContainer">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="cursor">
       <cursorShape>OpenHandCursor</cursorShape>
      </property>
      <property name="frameShape">
       <enum>QFrame::Panel</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>671</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuViews">
    <property name="title">
     <string>Views</string>
    </property>
    <addaction name="actionNewPlane"/>
    <addaction name="actionShowThisInOthers"/>
    <addaction name="actionShowOthersHere"/>
   </widget>
   <addaction name="menuViews"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNewPlane">
   <property name="text">
    <string>New slice view</string>
   </property>
  </action>
  <action name="actionShowThisInOthers">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>show this in others</string>
   </property>
  </action>
  <action name="actionShowOthersHere">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>show others here</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
